packages/pluggableWidgets/video-player-web/src/VideoPlayer.xml

```diff
@@ -8,15 +8,13 @@
     <properties>
         <propertyGroup caption="General">
             <propertyGroup caption="Data source">
-                <property key="urlExpression" type="expression">
+                <property key="videoUrl" type="textTemplate">
```

Comment:
Renaming it will cause breaking changes and loosing the values while updating the widget

Reply:
One question, if we maintain the type changing and doesn't change the key, that means no breaking changes?